### ISSUE TITLE
no rendering

### ISSUE THREAD

**Issue by a6d6061ed1** [ImpactDevelopment/ImpactClient#1307]
**Is your feature request related to a problem? Please describe.**
snow too bright, minecarts too lagy, hand obscuring my view

**Describe the solution you'd like**
add mod that'd prevent rendering of snow, minecarts and player's hand in 1st person and more that I didnt mention but are useful

**Comment by 0-x-2-2** [comment]
One suggestion/bug per issue.

**Comment by a6d6061ed1** [comment]
> One suggestion/bug per issue.

hm? it's one mod with different sub-options. should I really make 3 different suggestions for this?

**Comment by 0-x-2-2** [comment]
v

**Comment by ZeroMemes** [comment]
>One suggestion/bug per issue.

:clown_face:

**Comment by 0-x-2-2** [comment]
Can you not use a resource pack to do this?
Don't this would make minecarts any less laggy. 
Can you not use a resource pack to do this? Or hit the hide hud button. 🤔

**Comment by a6d6061ed1** [comment]
> Can you not use a resource pack to do this?

Yes I could use resource pack but having mod for this is much better as I can turn it off and on much easier and most importantly I dont have to make custom resource pack. Why is xray available in impact if it can be achieved with custom resource pack as well?

> Don't this would make minecarts any less laggy.

You're right but it'd be still useful for stacking a lot of minecarts in one place

> Can you not use a resource pack to do this? Or hit the hide hud button. 🤔

Hiding hud obviously hides entire hud instead of just player's hand

**Comment by 0-x-2-2** [comment]
> Why is xray available in impact if it can be achieved with custom resource pack as well?

xray texture packs don't work very well since 1.8 added cave culling.

**Comment by a6d6061ed1** [comment]
> xray resource packs don't work very well since 1.8 added cave culling.
this was just an example I hope you got the idea of what I meant by that

> your suggesting disabling the rendering therefor the crosshair will still interact with them.
Couldn't this be expanded with remove option like impact has right now for item entities? So they don't get spawned at all?
